install_config/topics/aws_service_account.adoc

```diff
@@ -0,0 +1,117 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_aws.adoc
+////
+
+== Configuring authorization for Amazon Web Services (AWS)
+
+*Permissions*
+AWS instances require either IAM account with Programmatic Access using an access and secret key or IAM role
+assigned to instances at creation to be able to request and manage load balancers and storage
+in {product-title}.
+
+The IAM account or IAM role must have the following policy permissions
+xref:..//install_config/configuring_aws.html#configuring-aws-permissions[permissions]
```

Comment:
xref:../install_config/configuring_aws.adoc#configuring-aws-permissions[permissions]

(extra / in the path and .html needs to be .adoc)